dragonmapper/html.py

```diff
@@ -0,0 +1,196 @@
+# -*- coding: utf-8 -*-
+"""Formatting Chinese into HTML with dragonmapper's functions"""
+
+
+from __future__ import unicode_literals
+from dragonmapper import hanzi
+from dragonmapper import transcriptions as trans
+from zhon import pinyin
+from zhon import zhuyin
+
+
+"""See recomended CSS style: DRAGONMAPPER_DIR/style.css"""
+
+
+_indentation = 0
+_line_html = ''
+_puctuation = ['，', '。', '“', '”', '：', '；', '？']
+_tones_marks = ['¯', 'ˊ', 'ˇ', 'ˋ', '˙', '1', '2', '3', '4', '5']
+
+
+def _identify(s):
+
+    """
+    Returns string of text type for HTML/CSS.
+
+    *s* is the string to identify.
+    """
+
+    if hanzi.has_chinese(s):
+        return "hanzi"
+    elif s in _puctuation:
+        return "punct"
+    elif s in _tones_marks:
+        return "tone-mark"
```

Comment:
It doesn't look like punctuation or tone marks ever come through this function. Can we get rid of those types?